workflow-varscan/varscan.wdl

```diff
@@ -168,32 +179,76 @@ output {
 }
 
 #=============================================================
-# Task for concatenating mplileups
+# Task for concatenating CNV and SNV variants
 #=============================================================
-task concatMpileup {
+task mergeVariantsNative {
 input {
  Array[File] filePaths
- Int jobMemory = 10
- Int timeout   = 20
+ String outputFile = "concatenated_variants"
+ String outputExtension = "csv"
+ Int jobMemory = 6
+ Int timeout   = 10
 }
 
 parameter_meta {
   filePaths: "Array of pileup files to concatenate"
   jobMemory: "memory in GB for this job"
+  outputExtension: "Extension of the output file"
+  outputFile: "Name of the output file"
   timeout: "Timeout in hours, needed to override imposed limits"
 }
 
 command <<<
```

Comment:
Can `set -euo pipefail` be added and grep exit code on non-match be handled or grep filter be changed to a tool that doesn't exit with non-zero on no match (e.g. sed -ne "/^chrom/p")

Reply:
used awk to handle this